fire/SFEquationsMod.F90

```diff
@@ -62,7 +68,11 @@ real(r8) function MaximumReactionVelocity(SAV)
       ! ARGUMENTS:
       real(r8), intent(in) :: SAV ! fuel surface area to volume ratio [/cm]
       
-      MaximumReactionVelocity = 1.0_r8/(0.0591_r8 + 2.926_r8*(SAV**(-1.5_r8)))
+      if (SAV < nearzero) then 
+        MaximumReactionVelocity = 0.0_r8
+      else 
+        MaximumReactionVelocity = 1.0_r8/(0.0591_r8 + 2.926_r8*(SAV**(-1.5_r8)))
+      end if
```

Comment:
I agree with the logic here because this prevents a singularity, but this change may cause the PR to break the bit-for-bit expectation in some edge cases, right?

Reply:
hmm well it is b4b right now?